### DISCUSSION TITLE
Daily 09/10

### DISCUSSION THREAD

**Discussion by BraianTroncoso** [grupo-ispc-2023-2-0/workflix-app#179] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
Ayer participe del que hicimos un meet y logramos avanzar con el tema back, estamos definiendo las siguientes issues y proyectando requirimientos junto con necesidades para el proyecto.
Hoy planeo hacer cuestionarios que se me estan quedando muy atras y ver alguna libreria o herramienta que podamos sumar para hacer alguna cosa mas simple o mejorar.

**Comment by NataliaAlvarezIspc** [comment]
El viernes estuve viendo videos sobre el  back. El sábado participé de la weekly, en donde definimos como seguir trabajando. El domíngo nos juntamos, también, para ver como se hace toda la conexión con el back. 
Hoy voy a hacer algunos cuestionarios que aún tengo pendientes en la plataforma y voy a intentar comenzar con el back de registración.

**Comment by marceloedelgado** [comment]
El fin de semana logré implementar el "Recycler View' para las tarjetas de los profesionales. El diseño de las tarjetas lo toma del archivo 'tarjeas_profesionales.xml' que trabaja con la Clase 'Profesional' y se sirve los datos de 'CatalogoActivity.java', también tuve que implementar un Adapter para que todo funcione.

Tuve problemas para redondear la parte inferior de los botones 'Contratar' pero igual quedo funcionando.

En el día de hoy comenzaré a ver la tarea que tengo por delante de crear una activity para ampliar la informaciön al tocar en el boton de una tarjeta del catalogo para que el usuario decidia contactarlo finalmente.

**Comment by Valentino399** [comment]
El sábado tuve practicando java con el tema de los booleanos y ahora sigo los tipos primitivos y hoy quiero hacer un par de cuestionarios que están dentro de la plataforma

**Comment by MarielaGomez** [comment]
Durante el finde semana logré poner en práctica android y realizar distintos proyectos para luego volcarlo al proyecto principal, también el Domingo estuvimos realizado un meet con el grupo de la facu para repasar la creación de tablas y su conexión con mysql
